internal/uri/uri.go

```diff
@@ -0,0 +1,14 @@
+package uri
```

Comment:
maybe `valid` is a better name, so we have `valid.Uris` (or `valid.URIs`) and `valid.Paths` ?

Reply:
Good idea.